### PR TITLE
Jscbc 831 named scopes first

### PR DESCRIPTION
**Pull request by osfameron**

### CODE REVIEW

File: modules/hello-world/pages/start-using-sdk.adoc [hunk 1]
```diff
@@ -39,7 +39,7 @@ Note: We have used the `-y` flag to take the initialization defaults. To change
 
 [source,json,indent=0]
 ----
-include::../examples/getting-started/package.json[tag=start-using-package-json]
+include::../examples/getting-started/package.json[tags=**]
```
Comment (maria-robobug) [inline modules/hello-world/pages/start-using-sdk.adoc:42]: Just a question for my benefit, what does `[tags=**]` do? 
I vaguely remember you mentioned Antora tags in the `package.json`, were they causing issues or just seemed odd?

File: modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc [hunk 2]
```diff
@@ -180,28 +182,28 @@ Here is the SDK 3 equivalent:
...
 
 == Serialization and Transcoding
 
-In SDK 2 the main method to control transcoding was through providing different `Document` instances (which in turn had their own transcoder associated), such as the `JsonDocument`. 
+In SDK 2 the main method to control transcoding was through providing different `Document` instances (which in turn had their own transcoder associated), such as the `JsonDocument`.
 This only worked for the KV APIs though -- Query, Search, Views, and other services exposed their JSON rows/hits in different ways.
 All of this has been unified in SDK 3 under a single concept: serializers and transcoders.
 
-By default, all KV APIs transcode to and from JSON -- you can also provide java POJOs which you couldn't in the past. 
+By default, all KV APIs transcode to and from JSON -- you can also provide java POJOs which you couldn't in the past.
```
Comment (maria-robobug) [inline modules/project-docs/pages/migrating-sdk-code-to-3.n.adoc]: `nodejs` instead of `java`.

**Review by maria-robobug — APPROVED** [review]
👍 LGTM, just minor comments added!

File: modules/hello-world/pages/start-using-sdk.adoc (hunk 1 above)
Comment (osfameron) [reply modules/hello-world/pages/start-using-sdk.adoc:42]: it shows everything, even if not tagged (as .json doesn't have comments so can't be tagged and remain valid)

**Pull request merged**